### PR TITLE
adds tangle website features section

### PR DESCRIPTION
**Pull request by vutuanlinh2k2**
## Summary of changes
_Provide a detailed description of proposed changes._
- 

### Proposed area of change
_Put an `x` in the boxes that apply._

- [ ] `apps/bridge-dapp`
- [ ] `apps/stats-dapp`
- [ ] `apps/webbsite`
- [ ] `apps/faucet`
- [x] `apps/tangle-website`
- [ ] `libs/webb-ui-components`

### Reference issue to close (if applicable)
_Specify any issues that can be closed from these changes (e.g. Closes #233)._
- Closes 

### Screen Recording
https://user-images.githubusercontent.com/69841784/230424730-02cda4ea-8daa-40f3-85df-d0e80a2020a5.mov

-----
### Code Checklist 
_Please be sure to add .stories documentation if any additions are made to `libs/webb-ui-components`._

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by devpavan04** [comment]
@vutuanlinh2k2 Please make the section responsive for tablet screens.

Screenshot 2023-04-10 at 8 19 10 AM

File: apps/tangle-website/src/styles/globals.css [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
 
-.sub-heading-1 {
-  @apply text-[16px] leading-[21.6px] /** small screen */
-  md:text-[18px] md:leading-[27px] /** medium screen */
-  font-medium;
+.section-description {
+  @apply text-[16px] leading-[25.6px] /** small screen */
+  md:text-[24px] md:leading-[40px] /** medium screen */
+  font-medium text-mono-140;
 }
```
Comment (AtelyPham) [inline apps/tangle-website/src/styles/globals.css:16]: Just ignore the font styles right now! We have a separate PR for adding the font styles in the webb-ui-kit.

**Review by AtelyPham — COMMENTED** [review]
I would like to suggest a few minor changes.

**Comment by dutterbutter** [comment]
Please resolve conflicts @vutuanlinh2k2

**Review by AtelyPham — APPROVED** [review]
LGTM 👍

**Review by AtelyPham — APPROVED** [review]
LGTM 👍

**Pull request merged**